package/features

```diff
@@ -32,3 +32,6 @@ packages 'mesh-batman-adv-15' \
 
 packages 'mesh-babel' \
 	'gluon-radvd'
+
+packages '!wireless-encryption-wpa3' \
+	'gluon-wireless-encryption-wpa2'
```

Comment:
How does this work? If I don't include the private wifi feature, will I have the `gluon-wireless-encryption-wpa2` package included by default?

Reply:
Exactly.